core/server/models/base/token.js

```diff
@@ -36,7 +36,7 @@ Basetoken = ghostBookshelf.Model.extend({
             .query('where', 'expires', '<', Date.now())
             .fetch(options)
             .then(function then(collection) {
-                collection.invokeThen('destroy', options);
+                return collection.invokeThen('destroy', options);
```

Comment:
I understand that only the return statement changed, I'm just questioning whether this code is even correct ?